python/paddle/fluid/dygraph/dygraph_to_static/ast_transformer.py

```diff
@@ -109,14 +108,21 @@ def transfer_from_node_type(self, node):
         self.visit(node.node)
 
         # Transform basic api of dygraph to static graph
-        BasicApiTransformer(node).ast_visit()
+        basic_api_trans = BasicApiTransformer(node)
+        basic_api_trans.ast_visit()
+        self.feed_name_to_arg_name = basic_api_trans._get_feed_name_to_arg_id()
```

Comment:
usually, name like "_xxx" is private method. Do you mind renaming the method? Such as "get_feed_name_to_arg_id"

Reply:
Thanks. I will modify it